automatic/waterfox/waterfox.nuspec

```diff
@@ -14,19 +14,16 @@
     <licenseUrl>https://github.com/MrAlex94/Waterfox/blob/master/LICENSE.md</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <projectSourceUrl>https://github.com/MrAlex94/Waterfox</projectSourceUrl>
-    <tags>waterfox admin web-browser firefox 64-bit</tags>
+    <tags>waterfox web-browser firefox 64-bit admin cross-platform foss</tags>
     <summary>Waterfox is basically a 64-bit version of Firefox.</summary>
     <description>
-## How and why Waterfox is here...
```

Comment:
Personally I don't find it strange at all.
Yeah, I stripped out `About Waterfox` heading beacuse that's what the description is all about, and that is the main header while this one is a sub header.
I would like to keep it.